### PR TITLE
Examples: prios based on user-interaction

### PR DESCRIPTION
**Pull request by addyosmani**
This was a user-case that came up from AMP folks.

### CODE REVIEW

**Comment by addyosmani** [comment]
Rebased as I just noticed it had some conflicts.

File: README.md [hunk 1]
```diff
@@ -40,6 +40,9 @@ want to avoid them interfering with e.g. loading of viewport images.
 medium/high priority, but developers may wish to load them all at low 
 priority. Similarly, developers may wish to load all first-party 
 resources that are critical with a high priority.
+* Resources fetched as a result of user-interaction often have varying
+priorities. Developers may wish to load these with a much higher priority
+as they can anticipate better exactly what the user needs.
```
Comment (yoavweiss) [inline README.md]: Maybe "When developers download a group of resources as a result of user interaction, those resources download priorities don't take into account the eventual usage and importance of those resources. Developers may wish to load these resources with priorities and dependencies which better represent their usage and the user's needs" ?

**Review by yoavweiss — COMMENTED** [review]
apologies... I thought I submitted my review, but it's just been sitting here for a while :/

**Comment by addyosmani** [comment]
@yoavweiss No worries at all. I just updated to your suggested text.

**Review by yoavweiss — APPROVED** [review]

**Pull request merged**